import-automation/executor/app/executor/import_executor.py

```diff
@@ -600,7 +627,7 @@ def _run_with_timeout(args: List[str],
         logging.exception(
             f'An unexpected exception was thrown: {e} when running {args}:'
             f' {message}')
-        return None
+        raise subprocess.TimeoutExpired(args, timeout)
```

Comment:
should it convert all exceptions to TimeoutExpired? 
Was this causing an error when testing manually?